site/faq.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Weave Net FAQ
+layout: default
+---
+
+
+
+###Q:How do I obtain the IP of a specific container when I'm using Weave?
+
+Use `weave ps` to see which address was allocated to a container on a Weave network.  
```

Comment:
Maybe `weave ps <container>` ?  (since that is a closer match to the question, and runs faster)